packages/toolpad-app/package.json

```diff
@@ -137,5 +137,8 @@
   "typings": "./index.d.ts",
   "publishConfig": {
     "access": "public"
+  },
+  "engines": {
+    "node": ">=18"
```

Comment:
Why so high?

```jsx
$ node --version
v16.16.0
```

Reply:
> Why so high?

We could treat it as any other dependency in the project, and keep it up to date through renovatebot.